python/apps/taiga/src/taiga/users/api/__init__.py

```diff
@@ -178,6 +183,37 @@ async def update_my_user(request: Request, form: UpdateUserValidator) -> User:
     )
 
 
+#####################################################################
+# delete user
+#####################################################################
+
+
+# TODO
+
+
+#####################################################################
+# delete info user
+#####################################################################
+
+
+@routes.my.get(
+    "/my/user/delete-info",
+    name="my.user.delete-info",
+    summary="Get user delete info",
+    response_model=UserDeleteInfoSerializer,
+    responses=ERROR_401,
+)
+async def get_user_delete_info(request: Request) -> UserDeleteInfoSerializer:
+    """
+    Get the delete info about a user.
```

Comment:
We should document a bit the endpoint, telling what is his endpoint about, when to use it, and what we return in the response. Specially considering this is not a classic CRUD endpoint.
Without this information, there isn't enough data to use this endpoint correctly.

Reply:
Done!